### PR TITLE
style(content): A bunch of small tweaks to get closer to spec

### PR DESCRIPTION
**Pull request by k88hudson**
@rlr @oyiptong @emtwo R?

### CODE REVIEW

**Comment by pdehaan** [comment]
Not sure why, but Travis-CI is reporting that we're using FF 31 again.
```sh
$ firefox -v
Mozilla Firefox 31.0
```
(ref: https://travis-ci.org/mozilla/activity-streams/builds/109784117#L1726-L1727)

File: content-src/components/ActivityFeed/ActivityFeed.scss [hunk 1]
```diff
@@ -61,10 +61,7 @@
     height: auto;
     left: 0;
     margin-left: -4px;
-    margin-top: -($feed-gutter / 2 + 18 / 2);
-    position: absolute;
-    top: 50%;
-    width: auto;
+    margin-top: -($feed-gutter-v / 2 + 18 / 2);
```
Comment (pdehaan) [inline content-src/components/ActivityFeed/ActivityFeed.scss]: Why the extra math of `18 / 2` vs just `9`?
Or should we make `18` some sort of variable if it has significance?

**Comment by rlr** [comment]
I can't get any data to display going straight to localhost or via the addon. Do I need to do something special/different?

The code changes lgtm.

**Comment by pdehaan** [comment]
Ditto, I'm getting the following output in Nightly (note the `E_TIMEOUT` error in the DevTools console):

nightly

**Comment by pdehaan** [comment]
I should point out that the `E_TIMEOUT` doesn't seem specific to this PR, I can see it in **master** today too (although **master** has pretty outline borders for missing items):

nightly

File: content-src/components/TopSites/TopSites.js [hunk 2]
```diff
@@ -12,14 +13,18 @@ const TopSites = React.createClass({
...
-        {sites.map((site) => (<a key={site.url} className="tile" href={site.url}>
-          <div className="tile-img-container">
-            <SiteIcon site={site} width={100} height={100} />
-          </div>
-          <div className="tile-title">
-            {site.provider_name}
-          </div>
-        </a>))}
+        {sites.map((site) => {
+          const color = site.favicon_colors && site.favicon_colors[0] && site.favicon_colors[0].color || [333, 333, 333];
```
Comment (pdehaan) [inline content-src/components/TopSites/TopSites.js]: Wait, isn't 255 == FF? Not sure what color `[333, 333, 333]` would be (whiter than white), or if these should be `[33, 33, 33]` (darker gray).

File: content-src/styles/variables.scss [hunk 3]
```diff
@@ -5,41 +5,58 @@ $dark-grey: #383E49;
...
-$tile-border: 1px solid #FFF;
+$tile-border: 1px #FFF;
 $tile-gutter-h: 20px;
-$tile-gutter-v: 20px;
-$tile-text-color: #5C5C5C;
-$tile-title-bg-color: #F2F2F2;
-$tile-title-hover-color: #333;
-$tile-title-hover-text-color: #FFF;
+$tile-gutter-v: 10px;
+$tile-title-bg-color: rgba(255, 255, 255, 0.8);
```
Comment (pdehaan) [inline content-src/styles/variables.scss]: Sass also lets you do fun shortcuts like `rgba($white, 0.8)`.

**Comment by pdehaan** [comment]
Needs rebase.

**Pull request merged**